pkg/operator/ceph/object/user/controller.go

```diff
@@ -152,6 +173,21 @@ func (c *ObjectStoreUserController) createUser(context *clusterd.Context, u *cep
 	}
 	objContext := object.NewContext(context, u.Spec.Store, u.Namespace)
 
+	// Wait with exponential backoff for the objectstore to be created
+	err := wait.ExponentialBackoff(backoff, func() (ok bool, err error) {
+		_, err = context.RookClientset.CephV1().CephObjectStores(u.Namespace).Get(u.Spec.Store, metav1.GetOptions{})
+		if err == nil {
+			return true, nil
+		}
+		// wait only if objectStore is not found. return on any other error
+		if errors.IsNotFound(err) {
+			return false, nil
```

Comment:
It would be good to add logging so we can see that the operator is waiting for the object store to exist

Reply:
ack